src/components/ResultList.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import cityImg from '../city.png'
+
+export interface City {
+    id: number,
+    ascii: string,
+    name: string
+}
+
+
+export const ResultList: React.FunctionComponent<{ cities: City[] }> = ({ cities }) => {
+    return <div className="ResultList">
+        <ol>
+            {cities.map((c) => <li key={c.id}>

```

Comment:
🤴: I would write `city` instead `c`.

```suggestion
            {cities.map((city) => <li key={city.id}>

```